From 0e66d8f81574b13402b7356bf8261739c4b8b90e Mon Sep 17 00:00:00 2001
From: Jan Macku <jamacku@redhat.com>
Date: Thu, 25 Apr 2024 15:00:33 +0200
Subject: [PATCH] ci: update actions/upload-artifact to `v4`

`v3` will be deprecated soon, so update to `v4`.

https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/

rhel-only

Related: RHEL-32494
---
 .github/workflows/gather-metadata.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.github/workflows/gather-metadata.yml b/.github/workflows/gather-metadata.yml
index f432f41811..08ad813971 100644
--- a/.github/workflows/gather-metadata.yml
+++ b/.github/workflows/gather-metadata.yml
@@ -22,7 +22,7 @@ jobs:
         uses: redhat-plumbers-in-action/gather-pull-request-metadata@v1
 
       - name: Upload artifact with gathered metadata
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           name: pr-metadata
           path: ${{ steps.Metadata.outputs.metadata-file }}
